src/sql/pg/create.sql

```diff
@@ -1,7 +1,7 @@
 create table if not exists "accounts" (
 "id" serial not null primary key,
 "name" varchar(255),
-"balance" float check (balance >= minimum_allowed_balance),
+"balance" float default 0 check (balance >= minimum_allowed_balance),
```

Comment:
Looks like a knex issue: https://github.com/tgriesser/knex/issues/662. Still looking into how to make it work.